projects/src/transmitter.cc

```diff
@@ -0,0 +1,157 @@
+#include "transmitter.h"
+#include "hal.h"
+#include "packet/serialize.h"
+#include "usbconfig.h"
+
+// This define can be useful when sizing message mailbox and memory pools
+#define ASSERT_MESSAGE_MEMORY_LIMIT FALSE
```

Comment:
We are already using chDbgAssert which will be stripped on release builds. What is the purpose of this extra layer?